src/main/java/com/googlecode/jsendnsca/StandardErrorExceptionHandler.java

```diff
@@ -0,0 +1,15 @@
+package com.googlecode.jsendnsca;
+
+/**
+ * {@link NonBlockingPassiveCheckSenderExceptionHandler} to print the handled exception to standard error.
+ *
+ * @author max.schwaab@gmail.com
+ */
+public class StandardErrorExceptionHandler implements NonBlockingPassiveCheckSenderExceptionHandler {
```

Comment:
Guess this can be made a private static inner class within the NonBlockingPassiveCheckSender as doesn't need to be a public top level class.